src/dynamics/solver/mod.rs

```diff
@@ -149,57 +149,24 @@ impl Plugin for SolverPlugin {
         substeps.add_systems(warm_start.in_set(SubstepSolverSet::WarmStart));
 
         // Solve velocities using a position bias.
-        substeps.add_systems(
-            (
-                |mut bodies: Query<RigidBodyQuery>,
-                 mut constraints: ResMut<ContactConstraints>,
-                 solver_config: Res<SolverConfig>,
-                 length_unit: Res<PhysicsLengthUnit>,
-                 time: Res<Time>| {
-                    solve_contacts(
-                        &mut bodies,
-                        &mut constraints.0,
-                        time.delta_seconds_adjusted(),
-                        1,
-                        true,
-                        solver_config.max_overlap_solve_speed * length_unit.0,
-                    );
-                },
-            )
-                .in_set(SubstepSolverSet::SolveConstraints),
-        );
+        substeps.add_systems(solve_contacts::<true>.in_set(SubstepSolverSet::SolveConstraints));
```

Comment:
This is an unrelated change, but I also finally decided to clean up the system registration here and made it use named systems instead of the mysterious anonymous closures that show up in traces.